app/models/early_career_payments/eligibility_matrix_calculator.rb

```diff
@@ -16,7 +9,7 @@ def initialize(eligibility)
     def eligible_later?
       return false if subject == :mathematics && itt_academic_year == "2018_2019"
```

Comment:
Also, didn't mention feels odd having mixture of symbols and strings here. These could be missing objects.